DiffusionPreprocessing/DiffPreprocPipeline_PostEddy.sh

```diff
@@ -353,6 +353,10 @@ main()
 		cp -p ${filename} ${to_location}
 	done
 
+  mkdir -p ${outdirT1w/qc
+  cp ${outdir}/qc/* ${outdirT1w}/qc
+  immv ${outdirT1w}/cnr_maps ${outdirT1w}/qc/cnr_maps
```

Comment:
Missing a closing bracket.  And, can we preserve the original time stamp on the files during the copy? And can we call the output directory "QC", just to match the convention that we are using for a "StructuralQC" directory?
i.e., this set of 3 lines then becomes:
```  
mkdir -p ${outdirT1w}/QC
cp -p ${outdir}/QC/* ${outdirT1w}/QC
immv ${outdirT1w}/cnr_maps ${outdirT1w}/QC/cnr_maps
```